versions/development/SPEC.md

```diff
@@ -1401,16 +1413,16 @@ task bwa_mem_tool {
   input {
     Int threads
     Int min_seed_length
-    Int min_std_max_min
+    Array[Int]+ min_std_max_min
```

Comment:
How did this ever work? Could you pass multiple values to a scalar `Int` in the past?

Reply:
i'm not sure, might have just been a typo?